test/api_container_stop_test.go

```diff
@@ -67,3 +68,31 @@ func (suite *APIContainerStopSuite) TestInvalidParam(c *check.C) {
 	//TODO
 	// 1. invalid timeout value
 }
+
```

Comment:
thanks for adding api test case,  it will be great to add more test for cli test  :)